config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  scope "(:locale)",locale: /en|vi/ do
+  scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
     get "/help", to: "static_pages#help"
+    get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
+    resources :users
```

Comment:
ở trên khai báo line 5 và 6 rồi thì ở đây bỏ bớt những routes thừa nha em
```suggestion
    resources :users, except: i%(new create)
```